docs/sources/get-started/index.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Learn how to get started with Explore Profiles
+keywords:
+  - Explore Profiles
+  - Concepts
+title: Get started with Explore Profiles
+menuTitle: Get started
+weight: 300
+---
+
+# Get started with Explore Profiles
+
+Profiles can help you identify errors in your apps and services.
+Using this information, you can optimize and streamline your apps.
+
+Your investigation begins with the big picture and then drills down using profile types, labels, and flame graphs to explore your data. To learn more, refer to [Concepts](../concepts/).
+
+![The Explore Profiles app default view](../images/explore-profiles-homescreen.png)
+
+## Before you begin
+
+To use Explore Profiles with Grafana Cloud, you need:
+
+* A Grafana Cloud account
+* A Grafana stack in Grafana Cloud with a configured Hosted profiles or Pyroscope data source
```

Comment:
Why is "Hosted" capitalized?

Reply:
Same as before. Hosted profiles is a data source name.